kratos/includes/model_part.h

```diff
@@ -568,6 +596,94 @@ class KRATOS_API(KRATOS_CORE) ModelPart : public DataValueContainer, public Flag
     }
 
 
+    ///@}
+    ///@name MasterSlaveConstraints
+    ///@{
+
+    SizeType NumberOfMasterSlaveConstraints() const
+    {
+        return mMasterSlaveConstraints.size();
+    }
+
+    MasterSlaveConstraintContainerType& MasterSlaveConstraints()
+    {
+        return mMasterSlaveConstraints;
+    }
+
+    void SetMasterSlaveConstraints(MasterSlaveConstraintContainerType::Pointer pOtherMasterSlaveConstraints)
```

Comment:
i would propose not to define this function. actually i would like to remove the similar one also for the Nodes, Elements and Conditions...(this last one is out of scope here, but just to motivate my comment)